tests/testutils/test_package_to_lint.py

```diff
@@ -0,0 +1,47 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+from pathlib import Path
+
+from pylint.testutils.primer import PRIMER_DIRECTORY_PATH, PackageToLint
+
+
+def test_package_to_lint() -> None:
+    """Test that the PackageToLint is instantiated correctly."""
+    expected_dir_path = PRIMER_DIRECTORY_PATH / "vimeo" / "graph-explorer"
+    expected_path_to_lint = expected_dir_path / "graph_explorer"
+    expected_pylintrc_path = expected_dir_path / ".pylintrcui"
+
+    package_to_lint = PackageToLint(
+        url="https://github.com/vimeo/graph-explorer.git",
+        branch="main",
+        directories=["graph_explorer"],
+        pylintrc_relpath=".pylintrcui",
+        pylint_additional_args=["--ignore-pattern='re*'"],
+    )
+    assert package_to_lint.url == "https://github.com/vimeo/graph-explorer.git"
+    assert package_to_lint.branch == "main"
+    assert package_to_lint.directories == ["graph_explorer"]
+    assert package_to_lint.pylintrc_relpath == ".pylintrcui"
+    assert package_to_lint.pylint_additional_args == ["--ignore-pattern='re*'"]
+    assert package_to_lint.paths_to_lint == [str(Path(expected_path_to_lint))]
+    assert package_to_lint.clone_directory == Path(str(Path(expected_dir_path)))
+    assert package_to_lint.pylintrc == Path(str(Path(expected_pylintrc_path)))
```

Comment:
Aren't these `Path` by default due to the refactor?
Anyway, `Path(str(Path` seems a bit redundant 😄

Reply:
Harg, you're right, and the 21mn for the pipeline really hurt for small thing like this.. 😓 Let's release 2.12 and focus on 2 primer batch right after that.

Reply:
We'll need to the `end_line` reporter first. We could make this `blocked` until that is merged?

Reply:
I'll merge and do another speed up MR before 2.12.0. This MR is already too big.